cmd/application.go

```diff
@@ -57,25 +55,55 @@ var templatesRefreshCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(applicationCmd)
 
+	applicationAddFlags(applicationGenerateCmd)
 	applicationCmd.AddCommand(applicationGenerateCmd)
 	applicationCmd.AddCommand(applicationPublishCmd)
 	applicationCmd.AddCommand(applicationTemplatesCmd)
 	applicationTemplatesCmd.AddCommand(templatesRefreshCmd)
 }
 
+var nameDefault = ""
+var descriptionDefault = "placeholder description, written to app.yaml"
+var templateDefault = ""
+var accessDefault = "private"
+
+func applicationAddFlags(cmd *cobra.Command) {
+	cmd.PersistentFlags().StringP("name", "n", nameDefault, "What's it called?")
```

Comment:
"what's it called" feels a little colloquial for a flag help string. IMHO questions are poor form in usage / help strings. 
```suggestion
	cmd.PersistentFlags().StringP("name", "n", nameDefault, "Name of the application")
```